app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,28 @@
   <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
 
   <%= csrf_meta_tags %>
+  
+  <!-- favicon for every device that needs it defined -->
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="./../../assets/favicon/apple-touch-icon-57x57.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="./../../assets/favicon/apple-touch-icon-114x114.png" />
```

Comment:
Hello @Kornil - 

Can you convert these to use the favicon_link_tag method?

http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-favicon_link_tag